login-workflow/src/screens/ForgotPasswordScreen/ForgotPasswordScreen.test.tsx

```diff
@@ -197,6 +197,7 @@ describe('Forgot Password Screen tests', () => {
         fireEvent.click(nextButton);
 
         const successMessage = screen.findByText('Success');
+        // eslint-disable-next-line
```

Comment:
Whats up with this? Can you just take the async await off of this?

Reply:
I tried but was happy with it. You can try if you want